data/scopes/defaults.ini

```diff
@@ -10,8 +10,7 @@ sangoma_states_mac_template = "sangoma-MAC_ADDRESS-states.tmpl"
 snom_general_template = "snom-general.tmpl"
 snom_model_mac_template = "snom-MODEL-MAC_ADDRESS.tmpl"
 snom_model_template = "snom-MODEL.tmpl"
-yealink_general_template = "yealink-y0.tmpl"
-yealink_model_mac_template = "yealink-MAC_ADDRESS.tmpl"
+yealink_template = "yealink.tmpl"
```

Comment:
- Use variable `tmpl_phone = ` for phone provisioning file
- Set `tmpl_phone` in the model scope